README.md

```diff
@@ -37,7 +37,12 @@ digest.write(b"123456789");
 assert_eq!(digest.sum16(), 0x906e);
 
 // with initial
-let mut digest = crc16::Digest::new_with_initial(crc16::X25, 0u16);
+let mut digest = crc16::Digest::new_with_initial(crc16::X25, 0xFFFF);
```

Comment:
@CLomanno this looks like a breaking change, no?
to get same result, the initial needs to be changed from `0u16` to `0xFFFF`; same applies to crc32/64

since we don't want to break folks, thoughts on keeping original behavior?

Reply:
gonna merge this for now given the other pr has been landed.
@CLomanno let me know your take on the breaking change

Reply:
@mrhooray you're right, this is a breaking change.  There are 2 ways I can see to fix this.
1) Make a different `update()` function that uses the original `!value` at the begining and end.
2) Make `rfl` a u8 and make a 3rd case that will handle backwards compatable `update()` calls.

I'll make the 2nd change and let you look at it.